### PR TITLE
A few fixes

### PR DESCRIPTION
**Pull request by yopdog**

### CODE REVIEW

**Comment by terrorist96** [comment]
Why change "where" to "who were"? 
Other changes are fine with me.

**Comment by yopdog** [comment]
Does this sentence make sense?  

>except for Agents of the Law Enforcement WHERE acting in pursuant to a Warrant.

**Comment by terrorist96** [comment]
Yes?

**Comment by yopdog** [comment]
ok, I will replace the word "where" with its definition from dictionary.reference.com so it's more clear:

>except for Agents of the Law Enforcement - in or at what place? - acting in pursuant to a Warrant.

??

**Comment by terrorist96** [comment]
in a position, case, etc., in which

At, to, or in any situation in which; wherever:

http://www.thefreedictionary.com/Where

It's correct. "who were" is correct too, but is apt for more casual speak as opposed to a legal constitution.

**Comment by yopdog** [comment]
Having looked it up, I see now you were right. I changed it back to "where".

File: Liberland-constitution.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The General Provisions shall constitute an integral part of the Constitution and
...
+* **§I.9.** For the purposes of this Constitution, the term “Person” shall signify any human being whatsoever; the term “Individual” shall signify a Person of age who does not lack mental capacity and is not incarcerated following a conviction in a court of law; the term “Agent of the Public Administration” shall signify any Person working for, or contractually bound with, any body of the Public Administration; the term “Member of the Public Administration” shall signify anyone who is holding a public office of one of the branches of the Public Administration; the term “Agent of the Law Enforcement” shall signify any Person working for, or acting in pursuant to a Warrant made public by, the Law Enforcement or the Office of the Public Prosecutor; the term “Citizen” shall signify anyone holding the Citizenship of the Free Republic of Liberland; the term “Resident” shall signify anyone within the jurisdiction of the Free Republic of Liberland who is not a Citizen.
```
Comment (yopdog) [inline Liberland-constitution.md]: Changed it to - or acting 'in' pursuant to a Warrant

File: Liberland-constitution.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
+*We, the Citizens of the Free Republic of Liberland, in order to secure Liberty, Life and Property, for ourselves and future generations, do ordain and establish the Constitution of the Free Republic of Liberland, attempting a harmonious consensus for the benefit of all those willing to assume responsibility over the course of their lives. Being aware of a long and shameful list of governments’ trespasses to the Rights of the sovereign Individuals, we hereby declare that the Public Administration governing the Free Republic of Liberland shall first and foremost respect the Bill of Rights and exercise only such functions as have been delegated to it under this Constitution. Therefore, we declare that whenever the Public Administration becomes an obstacle to, rather than a guarantor of, our Rights, it shall be our duty to alter or abolish such government, and to institute a new government for the restoration of the Rights which we consider inherent in all human beings.*
```
Comment (yopdog) [inline Liberland-constitution.md]: responsiblity --> responsibility

**Comment by terrorist96** [comment]
It should be pursuant, not in pursuant. In pursuance would also be correct, but we decided on pursuant, so no 'in' preceeding it. :)

**Comment by yopdog** [comment]
Ok, I removed 'in'

**Comment by KacperZajc** [comment]
can someone tell what is the change here?

**Comment by yopdog** [comment]
Just a few spelling fixes.

**Pull request merged**